packages/rn-tester/js/examples/Image/ImageExample.js

```diff
@@ -340,6 +346,199 @@ class MultipleSourcesExample extends React.Component<
   }
 }
 
+type LoadingIndicatorSourceExampleState = {|
+  imageHash: number,
+|};
+
+type LoadingIndicatorSourceExampleProps = $ReadOnly<{||}>;
+
+class LoadingIndicatorSourceExample extends React.Component<
```

Comment:
Is it possible to use hooks instead of classes?

Reply:
I made use of classes over functional components for the sake of consistency (as the other, previous examples also use classes).